src/components/icon/icon.svelte

```diff
@@ -14,11 +15,12 @@
 
   // Not actually used by the component, but used to preload SVGs.
   const svgCache = {}
-  export const preloadIcon = (name: string) =>
+  export const preloadIcon = (name: string) => {
     // Note: We do this in a |requestIdleCallback| because we want to do this as
     // soon as possible, but we want to make sure the consumer has a chance to
     // call setIconBasePath before we go and preload the icons (or we'll get a
     // 404).
+    if (isSSR) return
```

Comment:
nit: a new line may be helpful here.
```suggestion
    if (isSSR) return

```